detector_ros/src/detector_ros.cpp

```diff
@@ -0,0 +1,63 @@
+#include <detector_ros/detector_ros.hpp>
+
+namespace ariitk::detector_ros {
+
+int h_min,s_min,v_min,h_max,s_max,v_max,canny_lower,canny_upper,min_contour_area,canny_ker;
```

Comment:
don't define global variables, ever. put these inside init, and space them properly.